iped-parsers/iped-parsers-impl/src/main/java/dpf/mg/udi/gpinf/shareazaparser/ShareazaLibraryDatParser.java

```diff
@@ -165,28 +162,43 @@ private void storeSharedHashes(AlbumFolder folder, Map<Integer, LibraryFile> ind
         }
     }
 
-    private int countHashDBHits(LibraryFolder folder) {
+    private int countHashDBHits(LibraryFolders folders) {
+        Map<Integer, LibraryFile> indexToFile = folders.getIndexToFile();
+        int result = 0;
+        Set<Integer> indexesCounted = new HashSet<>();
+        for (LibraryFolder f : folders.getLibraryFolders())
+            result += countHashDBHits(f, indexesCounted);
+        countHashDBHits(folders.getAlbumRoot(), indexToFile, indexesCounted);
```

Comment:
Shoudn't line 171 be added to result? Not sure, but are album entries without library entries being counted?

Reply:
Tests look good to me, will merge this.